lib/get-client-config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = config => {
+  // Form https://github.com/npm/npm/blob/master/lib/cache/caching-client.js#L194
+  return {
+    proxy: {
+      http: config.get('proxy'),
+      https: config.get('https-proxy'),
+      localAddress: config.get('local-address'),
+    },
+    ssl: {
+      certificate: config.get('cert'),
+      key: config.get('key'),
+      ca: config.get('ca'),
+      strict: config.get('strict-ssl'),
+    },
+    retry: {
+      retries: parseInt(config.get('fetch-retries')),
+      factor: parseInt(config.get('fetch-retry-factor')),
+      minTimeout: parseInt(config.get('fetch-retry-mintimeout')),
+      maxTimeout: parseInt(config.get('fetch-retry-maxtimeout')),
+    },
+    userAgent: config.get('user-agent'),
+    defaultTag: config.get('tag'),
+    couchToken: config.get('_token'),
+    maxSockets: parseInt(config.get('maxsockets')),
```

Comment:
add radix argument to all `parseInt` statements. I’m surprised the linking doesn’t catch that. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Octal_interpretations_with_no_radix

Reply:
Yep.

There is an eslint rule for that: https://eslint.org/docs/rules/radix
But it's not enabled in `standard`: https://github.com/standard/eslint-config-standard/blob/master/eslintrc.json

This is one a reason I prefer other solution to `standard`, they enable most style rules and disable a lot of really useful rules to detect these type of bug...Anyway.

Reply:
Done